src/components/organisms/SchedulePageModal/SchedulePageModal.tsx

```diff
@@ -1,162 +1,118 @@
-import React, { useState, useMemo, FC } from "react";
```

Comment:
I don't think this component is even a modal is it? This is for the `Schedule` that is rendered in the `NavBar` yeah? Would it be better/more explicit to name it `NavBarSchedule` or similar?

Reply:
I didn't like the name of the component either. Suggested to rename it to `ScheduleModal` (in the PR description) but your name looks good too.
chosen yours: https://github.com/sparkletown/sparkle/pull/1302/commits/78137c7abb88cf26011ddff5462a8ae520d78df4